golos.referral/abi/golos.referral.abi

```diff
@@ -23,23 +63,28 @@
             ]
         },
         {
-           "name": "breakout_parametrs",
+           "name": "parameter",
            "base": "",
+           "fields": []
```

Comment:
TODO: make abigen smart enough to skip empty base types

Reply:
@zxcat And/or abidiff?

Reply:
why abidiff? abidiff should show such differences. furthermore, I think we can add binary abidiff to compare serialized abi-files. but not now.

Here I mean, there is no reason to inherit from empty struct in abi. it's just makes it slightly more complex (including serialization) without visible benefits.